### PR TITLE
make admin a main section

### PR DESCRIPTION
**Pull request by rberrelleza**
Part of #114

### CODE REVIEW

File: src/content/administration/cleanup.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+title: Cluster Cleanup
```
Comment (rberrelleza) [inline src/content/administration/cleanup.mdx:2]: @huguestennier is it possible to use <TierList> is the title of a page? (to mark that the entire feature only applies to a certain tier?)

File: src/content/administration/private-repositories/github-application.mdx [hunk 2]
```diff
@@ -1,35 +1,57 @@
...
+title: GitHub Application
 description: Use a GitHub Application so your users can access private repositories in their development environments
 sidebar_label: GitHub Application
 id: github-app
 ---
 
 import Image from '@theme/Image';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import TiersList from '@theme/TiersList';
```
Comment (huguestennier) [inline src/content/administration/private-repositories/github-application.mdx:11]:
```suggestion
import TiersList from '@theme/TiersList';

# GitHub Application<TiersList tiers="Scale Enterprise" />
```

File: docusaurus.config.js [hunk 3]
```diff
@@ -209,7 +209,28 @@ module.exports = {
...
+        to: '/administration/dashboard/',
+        from: ['/enterprise/administration/dashboard/']
+      },
+      {
+        to: '/administration/cleanup/',
+        from: ['/enterprise/administration/cleanup/']
+      },
+      {
+        to: '/administration/private-repositories/github-app/',
+        from: ['/enterprise/administration/private-repositories/github-app/']
```
Comment (pchico83) [inline docusaurus.config.js:223]: In SaaS, we are creating the GitHub App by ourselves. The environment is already integrated with a GitHub App from start. Should we move the GitHub App configuration doc under Self-Hosted?

**Review by pchico83 — APPROVED** [review]

File: docusaurus.config.js (hunk 3 above)
Comment (rberrelleza) [reply docusaurus.config.js:223]: Yeah, if it's only for Self-Hosted, I'll put the configuration part under self-hosted

**Pull request merged**